scrapper.py

```diff
@@ -89,9 +94,39 @@ def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r') as crawler_config_file:
+        crawler_config = json.load(crawler_config_file)
+
+    is_crawler_config_ok = True
+    if not isinstance(crawler_config, dict) or \
+            "base_urls" not in crawler_config or "total_articles_to_find_and_parse" not in crawler_config \
+            or "max_number_articles_to_get_from_one_seed" not in crawler_config:
+        is_crawler_config_ok = False
+        raise UnknownConfigError
+
+    elif not isinstance(crawler_config["base_urls"], list) or \
+            not (isinstance(url, str) for url in crawler_config["base_urls"]):
+        is_crawler_config_ok = False
+        raise IncorrectURLError
+
+    elif not isinstance(crawler_config["total_articles_to_find_and_parse"], int) \
```

Comment:
this one is surely long. Try to split these checks using more ifs for simplicity.